device/core/src/blob_upload/blob_upload_result_2.ts

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+'use strict';
+
+import { BlobUploadCommonResponseStub } from './../interfaces';
+
+// import {
+//   BlobUploadCommonResponse
+// } from "@azure/storage-blob"; // Change to "@azure/storage-blob" in your package
+
+
+export class BlobUploadResult2 {
```

Comment:
>BlobUploadResult2 [](start = 13, length = 17)

this seems to be the same class as BlobUploadResult so i don't see the need for this?